app/assets/stylesheets/application.css.scss

```diff
@@ -20,8 +20,14 @@ $neutral: #d9d9d9;
 
 @import "font-awesome";
 
+body {
+  background: #71868f;
+  font-weight: 300;
+  color: #f0f0f0;
```

Comment:
Color literals like `#f0f0f0` should only be used in variable declarations; they should be referred to via variable everywhere else.